test/test_corner_detection.jl

```diff
@@ -0,0 +1,73 @@
+function _detect_corners(file, n_corners)
+    img = load(file)
+    gry = img[1:1, :, :]
+    cv_n_corners = OpenCV.Size{Int32}(n_corners...)
+    _cv_corners = OpenCV.Mat(Array{Float32}(undef, 2, 1, prod(n_corners)))
```

Comment:
Out of curiosity, could this be declared `OpenCV.Mat(Matrix{Tuple{Float32,Float32}}(undef, n_corners))`? And then avoid the reshaping/`eachslice` below?

Reply:
Currently, `OpenCV.Mat` can only accept 3D arrays, not matrices:
```julia
  MethodError: no method matching OpenCV.Mat(::Matrix{Tuple{Float32, Float32}})
  The type `OpenCV.Mat` exists, but no method is defined for this combination of argument types when trying to construct it.
  
  Closest candidates are:
    OpenCV.Mat(::AbstractArray{T, 3}) where T<:Union{Float32, Float64, Int16, Int32, Int8, UInt16, UInt8}
     @ OpenCV ~/.julia/artifacts/dcc70861ed0ffc5427898f6aeb436620f33fa02f/OpenCV/src/Mat.jl:13
```
So that's not an option. But even if it was, the values in the input arrays are never mutated in-place after the function call, so I still need to deal with the output arrays, and their shape and form is as it is.

Reply:
You should be able to do
```julia
corners = Matrix{Tuple{Float32,Float32}}(undef, n_corners)
ret, cv_corners = OpenCV.findChessboardCorners(gry, cv_n_corners, OpenCV.Mat(reinterpret(reshape, Float32, corners)), 0)
@assert ret "Failed to detect any corners!"
return corners
```
if you think that's a little bit clearer/cleaner.